bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -220,9 +221,9 @@ export class PlotCanvasView extends BokehView
 
     if follow_enabled and has_bounds
       logger.warn('Follow enabled so bounds are unset.')
-      for xr in _.values(frame.x_ranges)
+      for xr in values(frame.x_ranges)
         xr.bounds = null
-      for yr in _.values(frame.y_ranges)
+      for yr in values(frame.y_ranges)
```

Comment:
The situation around `Object.keys()` and `Object.values()` sucks, because `keys()` is widely implemented and `values()` is not, so we have to provide the implementation for `values()`. I did consider adding a trivial wrapper `keys = Object.keys` to `core/util/array`.